packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -251,6 +256,20 @@ function renderMultipleTriggers(props = {}) {
   );
 }
 
+function ActionGroupTrigger() {
+  return (
+    <ActionGroup
+      selectionMode="single"
+      disallowEmptySelection
+      onSelectionChange={(sel) => console.log(sel)}>
+      <TooltipTrigger delay={0}>
+        <Item key="Key">Trigger Tooltip</Item>
+        <Tooltip>Tooltip is inside an ActionGroup</Tooltip>
+      </TooltipTrigger>
+    </ActionGroup>
+  );
+}
```

Comment:
Mind changing this from a console.log to something that changes a visible string in the story? Just worried that we won't overlook the console in manual testing in the future.

Maybe something like this:
```suggestion
function ActionGroupTrigger() {
  let [key, setKey] = useState('');
  return (
    <Flex direction="column">
      <div>Click the trigger to log the trigger's key:{key}</div>
      <ActionGroup
        selectionMode="single"
        disallowEmptySelection
        onSelectionChange={setKey}>
        <TooltipTrigger delay={0}>
          <Item key="Key">Trigger Tooltip</Item>
          <Tooltip>Tooltip is inside an ActionGroup</Tooltip>
        </TooltipTrigger>
      </ActionGroup>
    </Flex>
  );
}
```

Reply:
I implemented the suggestion.